pep-0596.rst

```diff
@@ -0,0 +1,191 @@
+PEP: 596
```

Comment:
You loose. Lukasz picked this number!
https://www.python.org/dev/peps/pep-0596/